client/client.go

```diff
@@ -76,21 +76,21 @@ type Client interface {
 	BuildQueue() ([]*model.Feed, error)
 
 	// BuildStart re-starts a stopped build.
-	BuildStart(string, string, int) (*model.Build, error)
+	BuildStart(string, string, int, []string) (*model.Build, error)
```

Comment:
maybe instead do `BuildStart(string, string, int, ...string)` so that it can remain an optional input parameter?